components/organisms/Layout.js

```diff
@@ -43,18 +43,21 @@ export const Layout = ({
         </a>
       </nav>
       <header>
+        <h2 className="sr-only">Main site header</h2>
```

Comment:
Nice work, I think these strings should be bilingual

Reply:
Message Isabelle Waltzing with what you would like translated, she is probably best to provide them.

Reply:
Thank you, I'll message her for the translate.